### DISCUSSION TITLE
How??

### DISCUSSION THREAD

**Discussion by InWorker1** [TBBOFBD/Titanium#18] [General]
How u make it bro. how did you start writing cheats? I really want to start, but I don't know how to do it. I study the language itself and would like to study the very essence of cheat programming in parallel. How do I do this? Give at least a couple of extensive tips

**Comment by matveitolochko** [comment]
Hey @InWorker1.
How did I make this: I just did.
How did I start writing cheats: I started with easier problems (Writing cheats for web games). You need to crawl before you can run ;)
You said you studied the language, do you mean rust? Because you don't need to do it in rust, you can also do it in python ;) (I checked your repos)
Again, start with something easier... I.E. web games
I am also writing a book on how to write cheats: [here](https://tbbofbd.github.io/Book/)
